lib/core/package.json

```diff
@@ -45,6 +45,7 @@
   },
   "peerDependencies": {
     "@storybook/builder-webpack5": "6.5.0-alpha.39",
+    "@storybook/manager-webpack5": "6.5.0-alpha.39",
```

Comment:
@gaetanmaisse says we can remove this part, because it's specified below, and yarn would be ok with that.

Reply:
Can we do this for all optional peer dependencies? If so, I think that could be an improvement